### DISCUSSION TITLE
Local playback stopping at the end of current track

### DISCUSSION THREAD

**Discussion by Huecuva** [jcorporation/myMPD#517] [Q&A] [answered]
Hi. So I'm having another strange issue with myMPD. I have local playback working but more often than not (it doesn't happen all the time, but it's rare that it doesn't) the local playback will stop playing at the end of whatever track is playing and then not start the next track even though I can see that the next track is playing on my mpd server. Indeed, even myMPD indicates that the main stream is playing, but local playback seems to just stop buffering at the end of the track and then stop playing when it runs out of buffer. On top of that, when it does this the browser tab needs to be refreshed with the buffer playing, otherwise it will just play what it already had buffered and then stop again. I refreshed the browser tab about 3 times before hitting the local playback button, only to have it play the previous song that mpd played because that's what was still in the myMPD buffer. It was only when the buffer was actually playing that I refreshed and it played the currently playing track when I clicked on local playback. It's like myMPD drops the stream connection or something, but the myMPD server and the mpd server are the same machine.

The server I have myMPD running on is a headless Debian VM with 4 CPU cores and 8GB of RAM. The machine I'm trying to play music on is my HTPC, with a core i7 and 16GB of RAM. This is all on my local gigabit LAN connection. There should be no hardware or network issues. I should probably also add that `Single` is not enabled in Settings.

I hope that all makes sense and the issue I'm having is understandable. It's very annoying when I'm trying to listen to music and it keeps stopping when I'm busy. 

If anyone has any ideas about what could be going on or how to resolve it, please let me know.

Cheers.

**Comment by jcorporation** [comment, marked as answer]
myMPD itself does not buffer the stream. The browser is buffering the stream. Do you test with different browsers?

**Reply by Huecuva** [reply]
I'm using Firefox. I have not tried with a different browser, but I can do that. I will report back my findings.

**Reply by Huecuva** [reply]
It appears that Chromium does not have this problem. That's interesting.

Thanks for the help. However, on an unrelated note, why does the "go to playing song" button disappear when I tell the queue to list all tracks? Is that supposed to happen? It would be nice if it didn't. The page forward/back buttons also disappear from the top of the queue but not the bottom. That seems weird.

Another issue I noticed: When I first installed myMPD and got it working, the next time I tried to use it I had forgotten the password. I went through the password reset process and set a new password but it no longer requests a password on any machine on my LAN, even if it has never been logged into myMPD before.

**Reply by jcorporation** [reply]
> why does the "go to playing song" button disappear when I tell the queue to list all tracks?

that is a bug

> password 

myMPD has no login feature
